lib/Frontend/nvqpp/ConvertDecl.cpp

```diff
@@ -299,6 +299,12 @@ bool QuakeBridgeVisitor::VisitVarDecl(clang::VarDecl *x) {
 
       // Did this come from a stdvec init op? If not drop out
       auto stdVecInit = initVec.getDefiningOp<cc::StdvecInitOp>();
+      if (auto mx = initVec.getDefiningOp<quake::MxOp>())
```

Comment:
Since these conditionals don't have anything to do with the `stdVecInit` and its subsequent `if(!stdVecInit)` I think you might want to move these lines above that `stdVecInit` allocation.

Reply:
Yes, will change.

Reply:
That's good to know. For now I just moved the code per Alex's suggestion.